lib/ansible/module_utils/facts/hardware/openbsd.py

```diff
@@ -119,13 +112,23 @@ def get_memory_facts(self):
         return memory_facts
 
     def get_uptime_facts(self):
-        uptime_facts = {}
-        uptime_seconds = self.sysctl['kern.boottime']
+        # On openbsd, we need to call it with -n to get this value as an int.
+        sysctl_cmd = self.module.get_bin_path('sysctl')
+        cmd = [sysctl_cmd, '-n', 'kern.boottime']
 
-        # uptime = $current_time - $boot_time
-        uptime_facts['uptime_seconds'] = int(time.time() - int(uptime_seconds))
+        rc, out, err = self.module.run_command(cmd)
+
+        if rc != 0:
+            return {}
 
-        return uptime_facts
+        uptime_seconds = out.strip()
```

Comment:
Perhaps this would be more clear named `kern_boottime`? It isn't really uptime seconds until after taking into account the current time as is done below.